src/components/tables/CurrentJobsTable.tsx

```diff
@@ -81,35 +87,46 @@ const CurrentJobsTable = () => {
     () =>
       [
         {
-          name: "title",
-          type: "text-long",
-          modifier: (data) => data.content.title,
+          name: "Talk Title",
```

Comment:
```suggestion
          name: "Title",
```
Let's restore to title please

Reply:
Alright.